src/libs/SearchUtils.ts

```diff
@@ -273,6 +273,19 @@ function getSortedTransactionData(data: TransactionListItemType[], sortBy?: Sear
     });
 }
 
+function getSortedReportData(data: ReportListItemType[]) {
```

Comment:
Hmm I'm not sure that there's much benefit in reusing that method. We would still need to check for undefined strings since the type is strictly defined as `string` and report.created can be undefined. After that, all that's left is the `localeCompare` vs `compareStringDates`, so I'll just keep `localeCompare` as a native function

Reply:
The dates are stored as string, so it should work fine here.